clamav-notify/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3-alpine
+
+ENV CLAM_VERSION=0.102.1-r0
+
+WORKDIR /clam
+
+RUN apk add --no-cache bash wget shadow lighttpd \ 
+  clamav=$CLAM_VERSION clamav-libunrar=$CLAM_VERSION
+
+RUN adduser -S -g clamav -u 1000 clam \
+    && chown -R clam:clamav /clam /var/log/clamav
```

Comment:
`/var/log/clamav` folder: does the clamav process not log to stdout? If not, it should be done imo

Reply:
Clamav does log to stdout, but this is done via configuration later. The default behaviour of this image when run directly is to log to the folder and run freshclam and clamd in parallel and then I override that via helm